packages/rpc-core/test/e2e/krummelanke.test.ts

```diff
@@ -2,43 +2,48 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import storage from '@polkadot/storage';
+import { Balance, SignedBlock, StorageChangeSet } from '@polkadot/types';
+import storage from '@polkadot/storage/static';
 import WsProvider from '@polkadot/rpc-provider/ws';
 
 import Rpc from '../../src';
 
+const randomAccount = '5HTqyWJHAVUieZnpb1V8gK4T1E4mnhkrUVSSzWBQd6kYgsVJ';
+const ALICE = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY';
+
 describe.skip('e2e krumme lanke', () => {
-  let api;
+  let api: Rpc;
 
   beforeEach(() => {
     jest.setTimeout(30000);
     api = new Rpc(new WsProvider('ws://127.0.0.1:9944'));
   });
 
   it('subscribes to storage', (done) => {
-    api.state
+    return (
+      api.state
       .subscribeStorage(
         [
-          [storage.system.accountNonce, '5Ejbye9R8ygByQPrDSasaUid1munedPZUmg4f118HGmtodGp'],
-          [storage.balances.freeBalance, '5DTestUPts3kjeXSTMyerHihn1uwMfLj8vU8sqF7qYrFacT7']
+          [storage.system.accountNonce, randomAccount],
+          [storage.balances.freeBalance, ALICE]
         ],
-        (data) => {
+        (data: StorageChangeSet) => {
           expect(data).toHaveLength(2);
-          expect(data[0].toNumber()).toEqual(0);
-          expect(data[1].toNumber()).not.toEqual(0);
+          expect([data as any][0].toNumber()).toEqual(0);
+          expect([data as any][1]).toBeInstanceOf(Balance);
+          expect([data as any][1].toNumber()).not.toEqual(0);
```

Comment:
Any better ideas here?

Reply:
I would drop this one completely - we don't have Krumme Lanke anymore. (Or at least rename to be current and point to an Alexander RPC)

As for the `StorageChangeSet`, this shows how old there are. So https://github.com/polkadot-js/api/blob/master/packages/types/src/rpc/StorageChangeSet.ts has the `.changes` part which this needs to query. 

So basically, something like (typing out of the top of my head), since changes in a option -

```js
// could also unwrap and test the result
expect(data.changes[0].value.isSome).toBe(true);
```

Reply:
Hmm, but the tests are actually passing and subscribing to these two
```
[storage.system.accountNonce, randomAccount],
[storage.balances.freeBalance, ALICE]
```
I'm getting back a subscription that is returning an array of type `[Nonce, Balance]` with the correct results?!

Reply:
Ahh, yes, you are correct, I am wrong - rpc does return an Array. (I should know _very_ well - not sure how I "forgot" that rpc-core mangles the storage return)

Since it returns an `Array<Codec>` it is not each, not great to cast it to a Tuple.